app/routes/lending/LendableObjectDetail.tsx

```diff
@@ -0,0 +1,118 @@
+import dayGridPlugin from '@fullcalendar/daygrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import FullCalendar from '@fullcalendar/react';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import moment from 'moment-timezone';
+import { useState } from 'react';
+import { Field } from 'react-final-form';
+import { Helmet } from 'react-helmet-async';
+import { useParams } from 'react-router-dom';
+import { Content } from 'app/components/Content';
+import { Button, TextArea, TextInput } from 'app/components/Form';
+import LegoFinalForm from 'app/components/Form/LegoFinalForm';
+import Modal from 'app/components/Modal';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import type { DetailedLendableObject } from 'app/store/models/LendableObject';
+import { createValidator, required } from 'app/utils/validation';
+
+type Params = {
+  lendableObjectId: string;
+};
+
+const LendableObjectDetail = () => {
+  const { lendableObjectId } = useParams<Params>();
+  const [showLendingForm, setShowLendingForm] = useState(false);
+  const [startString, setStartString] = useState('');
+  const [endString, setEndString] = useState('');
```

Comment:
```suggestion
  const [start, setStart] = useState('');
  const [end, setEnd] = useState('');
```
why double type it

Also, what is start and end 🤨

Reply:
The start and end is the period they want to lend the object. The user can mark an area of the calender and then a form for requesting to lend the object pops up, where the start and end of the ledning period is already filled out. It's done this way with the intention that the periods that the object is already lended to someone else can be blocked out in the calender, so it's impossible to request to lend an object on dates it's already busy (to spare those who will review the ledning request a bit of work)